pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTopics.java

```diff
@@ -563,10 +563,14 @@ void run() throws PulsarAdminException {
         + ", locking required.")
         private boolean subscriptionBacklogSize = false;
 
+        @Parameter(names = { "-etb",
+                "--get-earliest-time-in-backlog" }, description = "Set true to get earliest time in backlog")
```

Comment:
```suggestion
                "--get-backlog-in-mills" }, description = "Set true to get earliest time in backlog")
```

Reply:
Can `--get-backlog-in-mills` reflect that it will get a timestamp? Maybe it's a little confusing as it looks like getting the backlog itself. (It's just my question and I can still change it to `--get-backlog-in-mills`)

Reply:
Ok, to avoid confusion, I think we can only return the timestamp of the earliest unacked message?

Reply:
> Ok, to avoid confusion, I think we can only return the timestamp of the earliest unacked message?

OK, I'll do it.

Reply:
> Ok, to avoid confusion, I think we can only return the timestamp of the earliest unacked message?

Done